src/index.js

```diff
@@ -68,6 +94,16 @@ function is(x, type) {
     return false
 }
 
+/**
+ * Often, we don't really care whether the object is a certain type,
+ * we just want to know whether we can treat it like a certain type.
+ * Use like(subject, type) to test whether a subject adheres to the same interface as is defined by type
+ * Example: like(looker, 'object')
+ *
```

Comment:
`like` with `'object'` as second parameter is not the intended use. 
Instead maybe change to:

Example:  like(viewer, Looker)  // true

This 're-uses' the `viewer` object which is mentioned in the docs. Maybe it's better to make a real example:

```js
const Looker = mix(superclass => class Looker extends superclass {
  look() {}
})
var viewer = {
  look() {} // same interface as Looker
}
is(viewer, Looker)    // false, but
like(viewer, Looker)  // true
```